portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,9 +73,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             return;
         }
         for(int i = 0; i < numberOfCommentsToDisplay; i++){
+
```

Comment:
Nit: remove leading empty line at the beginning of a scope?

Reply:
Resolved